assets/app/scripts/directives/logViewer.js

```diff
@@ -71,8 +71,8 @@ angular.module('openshiftConsole')
 
 
 
-
-            var updateScrollLinks = function() {
+            // is just toggling show/hide, nothing else.
+            var toggleScrollLinks = function() {
```

Comment:
Ah... true, it sets `showScrollLinks` to `true || false`, but via calculation.  hmm.  `setScrollLinksVisibility`?  Dunno if i like that either...